### DISCUSSION TITLE
[s3] Signature version "s3v4" is unsupported

### DISCUSSION THREAD

**Discussion by nsteinmetz** [grafana/mimir#2162] [Help and support] [answered]
#### Describe the bug

When using OVH Object storage in S3 mode, we have to set signature_version: s3v4

Cf https://docs.ovh.com/gb/en/storage/getting_started_with_the_swift_S3_API/#configure-aws-client

```
[plugins]
endpoint = awscli_plugin_endpoint

[profile default]
region = <region>
s3 =
  endpoint_url = https://s3.<region>.cloud.ovh.net
  signature_version = s3v4
s3api =
  endpoint_url = https://s3.<region>.cloud.ovh.net
```

However, when settings this version, mimir reports it's 

```
Jun 21 04:08:38 crnt-d10-sys1 mimir[24636]: error validating config: invalid rulestore config: unsupported signature version
```

#### To Reproduce

1. Start Mimir 2.1.0
2. Try to set `signature_version: s3v4`
3. Restart Mimir

#### Expected behavior

S3 connection should work as expected.

#### Environment

- Infrastructure: bare-metal
- Deployment tool: manual

**Comment by dimitarvdimitrov** [comment]
Mimir supports `v4`. Is `s3v4` different from `v4`?

**Comment by nsteinmetz** [comment, marked as answer]
Hi,

Indeed it works as is - I was spotted by another bug once I enabled multi-tenancy. So it took me some time to disctinct both issues.

so `v4` works, no need for `s3v4`
